alpha/apps/kaltura/lib/kPermissionManager.php

```diff
@@ -702,6 +706,96 @@ private static function initPermissionsMap()
 				// merge current role map to the global map
 				self::$map = array_merge_recursive(self::$map, $roleMap);
 			}
+
+			self::removeLimitedPermissions();
```

Comment:
why we only remove permissions on this case?
on line 697 we extract user associated role and permissions by calling it with null